### PR TITLE
Translate Manipulating the DOM with Refs page

### LINKED ISSUE #388: Manipulating the DOM with Refs
Page: https://id.react.dev/learn/manipulating-the-dom-with-refs

Assignee: @andriawan

### PR DESCRIPTION
**Pull request by andriawan**
Closes #388 

### Description

This PR is aimed at resolving the task of translating the "Manipulating the DOM with Refs" page.

### CODE REVIEW

**Comment by r17x** [comment]
Any updates @andriawan

**Comment by andriawan** [comment]
> Any updates @andriawan

Sorry for slow progress...kind of busy at main work... 😅  
promise i will finish the translation this week @r17x

**Comment by andriawan** [comment]
this PR is ready for review. cc @r17x @zainfathoni @mhaidarhanif @resir014 @mazipan

File: src/content/learn/manipulating-the-dom-with-refs.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
 ---
-title: 'Manipulating the DOM with Refs'
+title: 'Memanipulasi DOM dengan Refs'
 ---
 
 <Intro>
 
-React automatically updates the [DOM](https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Introduction) to match your render output, so your components won't often need to manipulate it. However, sometimes you might need access to the DOM elements managed by React--for example, to focus a node, scroll to it, or measure its size and position. There is no built-in way to do those things in React, so you will need a *ref* to the DOM node.
+React secara otomatis memperbarui [DOM (*Document Object Model*)](https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Introduction) agar sesuai dengan keluaran *render*, sehingga komponen Anda tidak perlu sering memanipulasinya. Namun, terkadang Anda mungkin perlu mengakses elemen DOM yang dikelola oleh React--misalnya, untuk memberikan fokus pada sebuah *node*, menggulir ke sana, atau mengukur ukuran dan posisinya. Tidak ada cara bawaan untuk melakukan hal-hal tersebut di React, sehingga Anda memerlukan *ref* ke *node* DOM.
```
Comment (r17x) [inline src/content/learn/manipulating-the-dom-with-refs.md:7]:
```suggestion
React secara otomatis memperbarui [DOM (*Document Object Model*)](https://developer.mozilla.org/docs/Web/API/Document_Object_Model/Introduction) agar sesuai dengan keluaran *render*, sehingga komponen Anda tidak perlu sering memanipulasinya. Namun, terkadang Anda mungkin perlu mengakses elemen DOM yang dikelola oleh React--misalnya, untuk memberikan fokus pada sebuah *node*, menggulir ke sana, atau mengukur ukuran dan posisinya. Tidak ada cara bawaan untuk melakukan hal-hal tersebut di React, sehingga Anda memerlukan *ref* ke simpul DOM.
```

File: src/content/learn/manipulating-the-dom-with-refs.md [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
 
 </Intro>
 
 <YouWillLearn>
 
-- How to access a DOM node managed by React with the `ref` attribute
-- How the `ref` JSX attribute relates to the `useRef` Hook
-- How to access another component's DOM node
-- In which cases it's safe to modify the DOM managed by React
+- Bagaimana cara mengakses sebuah *node* DOM yang diatur React dengan atribut `ref`
```
Comment (r17x) [inline src/content/learn/manipulating-the-dom-with-refs.md:13]:
```suggestion
- Bagaimana cara mengakses sebuah simpul DOM yang diatur React dengan atribut `ref`
```

bisa juga menggunakan `simpul (*node*)`

File: src/content/learn/manipulating-the-dom-with-refs.md [hunk 3]
```diff
@@ -1,52 +1,52 @@
...
+- Bagaimana atribut JSX `ref` berelasi dengan Hook `useRef`
+- Bagaimana cara mengakses *node* DOM dari komponen lain
```
Comment (r17x) [inline src/content/learn/manipulating-the-dom-with-refs.md:15]: Idem

File: src/content/learn/manipulating-the-dom-with-refs.md [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
+- Dalam kasus seperti apa memodifikasi DOM yang diatur React dengan aman.
 
 </YouWillLearn>
 
-## Getting a ref to the node {/*getting-a-ref-to-the-node*/}
+## Mendapatkan sebuah ref untuk node {/*getting-a-ref-to-the-node*/}
 
-To access a DOM node managed by React, first, import the `useRef` Hook:
+Untuk mengakses sebuah *node* DOM yang diatur React, pertama, impor Hook `useRef`:
```
Comment (r17x) [inline src/content/learn/manipulating-the-dom-with-refs.md:22]: idem

File: src/content/learn/manipulating-the-dom-with-refs.md [hunk 5]
```diff
@@ -1,52 +1,52 @@
...
 
-Then, use it to declare a ref inside your component:
+Kemudian, deklarasikan *ref* di dalam komponen Anda:
 
 ```js
 const myRef = useRef(null);
 ```
 
-Finally, pass it to the DOM node as the `ref` attribute:
+Terakhir, oper ke DOM node sebagai atribut `ref`:
```
Comment (r17x) [inline src/content/learn/manipulating-the-dom-with-refs.md:34]: idem dan beberapa line kebawah

**Review by r17x — CHANGES_REQUESTED** [review]

**Comment by andriawan** [comment]
been already improved translation `node` in the latest commit. cc @r17x

**Review by r17x — APPROVED** [review]
Thanks for your contribution @andriawan 🚀

**Pull request merged**